test/bddtests/config_steps.go

```diff
@@ -0,0 +1,238 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package bddtests
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/DATA-DOG/godog"
+	"github.com/pkg/errors"
+	"github.com/trustbloc/fabric-peer-test-common/bddtests"
+)
+
+type configSteps struct {
+	BDDContext *bddtests.BDDContext
+}
+
+// NewConfigSteps returns ledger config steps
+func NewConfigSteps(context *bddtests.BDDContext) *configSteps {
+	return &configSteps{BDDContext: context}
+}
+
+// DefineTransientCollectionConfig defines a new transient data collection configuration
```

Comment:
comment doesn't correspond to function

Reply:
I'll fix this in my next PR. It took 3 hours to get the CI to pass on this one :)

Reply:
I know, that's why I approved right away. It is not important comment anyway.